File: Arrays & Hashing/217. Contains Duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        count={}
        for i in nums:
            if i in count:
                return True
            else:
                count[i]=1
        return False
        

File: Arrays & Hashing/217. Contains Duplicate.cpp
#include <unordered_set>
#include <vector>
using namespace std;

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> duplicate;
        for (int i:nums) {
            if (duplicate.count(i)){
                return true;
            }
            duplicate.insert(i);
        }
        return false;
    }
};